python/pyspark/sql/tests.py

```diff
@@ -558,6 +558,18 @@ def test_create_dataframe_from_objects(self):
         self.assertEqual(df.dtypes, [("key", "bigint"), ("value", "string")])
         self.assertEqual(df.first(), Row(key=1, value="1"))
 
+    def test_to_localiterator_for_dataframe(self):
```

Comment:
Oh, the jira reported two test cases for rdd and dataframe. But yes, I think we can remove this.